projects/ghostscript/gstoraster_fuzzlib.h

```diff
@@ -48,17 +48,25 @@ static int gs_stdnull(void *inst, const char *buf, int len)
 	return len;
 }
 
-int gs_to_raster_fuzz(const unsigned char *buf, size_t size, int color_scheme)
+int gs_to_raster_fuzz(
```

Comment:
nit: this is now more generic, testing codepaths that are not used by gstoraster.  This should probably be named fuzz_gs_device now (or similar).